kubernetes/redpanda/values.yaml

```diff
@@ -10,14 +10,51 @@ image:
   # Overrides the image tag whose default is the chart appVersion.
   tag: ""
 
-redpanda:
-  advertised_rpc_api:
-    port: 33145
-  advertised_kafka_api:
-    port: 9092
-  admin:
-    port: 9644
-  smp: 1
+config:
+  config_file: /etc/redpanda/redpanda.yaml
+  license_key: ""
+  node_uuid: mf1PqvLg826mCjiYRRKcBb4MjtL88agNyGajk3GoELmgKixYA
```

Comment:
this should be empty or fail. node_uuid *must* be unique. what' sthe point of `uuid` if it's the same.

Reply:
i'm actually confused by this file altogheter. can you write a comment on top

Reply:
OK, I'll generate a UUID, and also write a comment for values.yaml.